tap_bing_ads/__init__.py

```diff
@@ -54,6 +58,39 @@
 
 ARRAY_TYPE_REGEX = r'ArrayOf([A-Za-z0-9]+)'
 
+def should_retry_httperror(exception):
+    """ Return true if exception is required to retry otherwise return false """
+    try:
+        if isinstance(exception, ConnectionError) or isinstance(exception, ssl.SSLError):
+            return True
+        elif isinstance(exception, suds.transport.TransportError) or isinstance(exception, socket.timeout):
+            return True
+        elif type(exception) == URLError:
+            return True
+        elif type(exception) == Exception and exception.args[0][0] == 408:
```

Comment:
Please add what is 408 and why we are retrying in the comment.

Reply:
A 408 Request Timeout is an HTTP response status code that indicates the server didn't receive a complete request message within the server's allotted timeout period. A suds SDK catches HTTPError with status code 408 and raises Exception: (408, 'Request Timeout'). That's why we retrying this error also. Added comment in the code, also.